apps/hash-frontend/next.config.js

```diff
@@ -1,10 +1,11 @@
 const { config } = require("dotenv-flow");
 const withTM = require("next-transpile-modules")([
   "@hashintel/design-system",
+  "@local/advanced-types",
   "@local/hash-graph-client",
   "@local/hash-graphql-shared",
   "@local/hash-isomorphic-utils",
-  "@local/hash-subgraph",
+  "@local/hash-subgraph/main",
```

Comment:
I don't think this is correct. Shouldn't it just be the package?
```suggestion
  "@local/hash-subgraph",
```

Otherwise I get the following error: `next-transpile-modules - an unexpected error happened when trying to resolve "@local/hash-subgraph/main"`